components/inputs/input-select-styles.js

```diff
@@ -23,73 +25,69 @@ export const selectStyles = css`
 		line-height: 1.2rem;
 		margin: 0;
 		max-height: calc(2rem + 2px);
+		outline: 1px solid var(--d2l-color-galena);
+		outline-offset: -1px;
+		padding-block: calc(0.4rem + 1px);
+		padding-inline: calc(0.75rem + 1px) calc(2px + 0.8rem + 1px + 11px + 16px + 1px);
 		vertical-align: middle;
 	}
 	:host([dir="rtl"]) .d2l-input-select {
 		background-position: center left 17px;
 	}
-	.d2l-input-select,
-	.d2l-input-select:hover:disabled {
-		background-color: #ffffff;
-		border-color: var(--d2l-color-galena);
-		border-width: 1px;
-		box-shadow: inset 0 2px 0 0 rgba(177, 185, 190, 0.2); /* corundum */
-		padding: 0.4rem 0.75rem;
-	}
-	.d2l-input-select,
-	.d2l-input-select:disabled,
-	.d2l-input-select:hover:disabled,
-	.d2l-input-select:focus:disabled {
-		background-image: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTEiIGhlaWdodD0iNyIgdmlld0JveD0iMCAwIDExIDciIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CiAgPHBhdGggZD0iTTEgMmw0LjUgNE0xMCAyTDUuNSA2IiBzdHJva2U9IiM1NjVBNUMiIHN0cm9rZS13aWR0aD0iMiIgZmlsbD0ibm9uZSIgZmlsbC1ydWxlPSJldmVub2RkIiBzdHJva2UtbGluZWNhcD0icm91bmQiLz4KPC9zdmc+");
-		padding-right: calc(2px + 0.8rem + 1px + 11px + 16px);
-	}
-	:host([dir="rtl"]) .d2l-input-select,
-	:host([dir="rtl"]) .d2l-input-select:disabled,
-	:host([dir="rtl"]) .d2l-input-select:hover:disabled,
-	:host([dir="rtl"]) .d2l-input-select:focus:disabled {
-		padding-left: calc(2px + 0.8rem + 1px + 11px + 16px);
-		padding-right: 0.75rem;
-	}
-	.d2l-input-select:hover,
-	.d2l-input-select:focus {
-		border-color: var(--d2l-color-celestine);
-		border-width: 2px;
-		outline-style: none; /* Safari */
-		outline-width: 0;
-		padding: calc(0.4rem - 1px) calc(0.75rem - 1px);
-		padding-right: calc(2px + 0.8rem + 1px + 11px + 16px - 1px);
-	}
-	:host([dir="rtl"]) .d2l-input-select:hover,
-	:host([dir="rtl"]) .d2l-input-select:focus {
-		padding-left: calc(2px + 0.8rem + 1px + 11px + 16px - 1px);
-		padding-right: calc(0.75rem - 1px);
+
+	.d2l-input-select:not([disabled]):hover,
+	.d2l-input-select:not([disabled]):focus {
```

Comment:
Does it make sense to use the `${unsafeCSS(getFocusPseudoClass())}` stuff here to limit the outline to `:focus-visible`?